src/iristilesource.js

```diff
@@ -87,33 +85,29 @@
       this._tileWidth = 256;
       this._tileHeight = 256;
 
-      this.tileSize = this._tileWidth;
-      this.tileOverlap = 0;
-
       const layers = data.extent.layers;
 
       const maxLayer = layers.length - 1;
       const maxScale = layers[maxLayer].scale;
-      this.width = data.extent.width * maxScale;
-      this.height = data.extent.height * maxScale;
+
+      this.width = parseInt(Math.ceil(data.extent.width * maxScale), 10);
```

Comment:
`parseInt(Math.ceil(x), 10)` seems very odd to my eyes. Why the parseInt? It's already a number because of the Math.ceil(), and even before that it should be a number since you're doing math on it.

It seems to me we can lose all of the parseInt calls throughout, unless there's one where we're converting a string to a number, and then the parseInt (or parseFloat) should be happening first rather than last.